akka-http-core/src/test/scala/akka/http/javadsl/JavaInitializationSpec.scala

```diff
@@ -1,66 +0,0 @@
-/*
- * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
- */
-
-package akka.http.javadsl
-
-import org.scalatest.{ Matchers, WordSpec }
-
-class JavaInitializationSpec extends WordSpec with Matchers {
-
-  implicit class HeaderCheck[T](self: T) {
-    def =!=(expected: String) = {
-      self should !==(null)
-      self.toString shouldBe expected
-    }
-  }
-
-  "EntityTagRange" should {
-    "initializes the right field" in {
-      akka.http.scaladsl.model.headers.EntityTagRange.`*` =!= "*"
-      akka.http.javadsl.model.headers.EntityTagRanges.ALL =!= "*"
```

Comment:
Shouldn't we keep testing the constants that still work?

Reply:
This was some crazy edge case so tests best to keep AFAICS